<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingDock</class>
 <widget class="QDockWidget" name="DrawingDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>561</width>
    <height>225</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/toolBar/可视化.png</normaloff>:/toolBar/可视化.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid;
border-color: rgb(100, 100, 100);</string>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>实时曲线</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_setting">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border:0px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../resource/BiVMS2020.qrc">
        <normaloff>:/界面/BiVMS/MainWnd/魔方1.png</normaloff>:/界面/BiVMS/MainWnd/魔方1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QWidget" name="widget_setting" native="true">
      <property name="styleSheet">
       <string notr="true">border:0px;</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;
color: rgb(85, 85, 255);</string>
             </property>
             <property name="text">
              <string>绘图窗口</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_showX">
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>X位移</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_showY">
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>Y位移</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_showZ">
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;</string>
             </property>
             <property name="text">
              <string>Z位移</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;
color: rgb(85, 85, 255);</string>
             </property>
             <property name="text">
              <string>显示时长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_count">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;
</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>59</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_time">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:0px;
font: 9pt &quot;微软雅黑&quot;;
</string>
             </property>
             <item>
              <property name="text">
               <string>秒</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>分</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>时</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidget">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/BiVMS2020.qrc"/>
 </resources>
 <connections/>
</ui>
